src/Grid/grid.jl

```diff
@@ -466,6 +523,59 @@ Returns all vertex sets of the grid.
 
 n_faces_per_cell(grid::Grid) = nfaces(getcelltype(grid))
 
+
+"""
+    getfacetset(grid::AbstractGrid, setname::String)
+
+Returns all facets in a `Set` of a given `setname`. 
+"""
+getfacetset(grid::AbstractGrid{sdim}, setname::String) where sdim = getfacetsets(grid)[setname]
+
+"""
+    getfacetsets(grid::AbstractGrid)
+
+Returns all facet sets of the grid.
+"""
+getfacetsets(grid::AbstractGrid{1}) = grid.vertexsets
+getfacetsets(grid::AbstractGrid{2}) = grid.edgesets
+getfacetsets(grid::AbstractGrid{3}) = grid.facesets
```

Comment:
Since facet refers to `rdim-1` this isn't always true then, right?
So it would be better to actually just store the `peaksets`, `ridgesets`, and `facetsets` directly?